pkg/cmd/pulumi/new.go

```diff
@@ -1112,3 +1117,29 @@ func containsWhiteSpace(value string) bool {
 	}
 	return false
 }
+
+func validateStackProjectName(b backend.Backend, stackName, projectName string) error {
```

Comment:
A comment explaining what the stackName and projectName arguments are would be helpful.
Specifically that they're the `--stack` and `--project` flag values,
and that stackName could be "$name" or "organization/$project/$name".